static/src/js/schemas/uiSchemas/collections/collectionInformation.js

```diff
@@ -69,6 +69,10 @@ const collectionInformationUiSchema = {
               'ui:row': [
                 {
                   'ui:col': {
+                    style: {
+                      padding: '1.25rem 1.5rem',
```

Comment:
Can we pull this styling out as a variable as we are using it a couple times here